includes/Admin/Settings_Screens/Commerce.php

```diff
@@ -49,6 +50,55 @@ public function __construct() {
 	 */
 	public function enqueue_assets() {
 
+		if ( Admin\Settings::PAGE_ID !== Framework\SV_WC_Helper::get_requested_value( 'page' ) || ( self::ID !== Framework\SV_WC_Helper::get_requested_value( 'tab' ) ) ) {
+			return;
+		}
+
+		wp_enqueue_script( 'facebook-for-woocommerce-settings-commerce', facebook_for_woocommerce()->get_plugin_url() . '/assets/js/admin/settings-commerce.min.js', [ 'facebook-for-woocommerce-modal', 'jquery-tiptip' ], \WC_Facebookcommerce::PLUGIN_VERSION );
+
+		wp_localize_script( 'facebook-for-woocommerce-settings-commerce', 'facebook_for_woocommerce_settings_commerce', [
+			'default_google_product_category_modal_message' => $this->get_default_google_product_category_modal_message(),
+			'default_google_product_category_modal_buttons' => $this->get_default_google_product_category_modal_buttons(),
+		] );
+	}
+
+
+	/**
+	 * Gets the message for Default Google Product Category modal.
+	 *
+	 * @since 2.1.0-dev.1
+	 *
+	 * @return string
+	 */
+	private function get_default_google_product_category_modal_message() {
+
+		return wp_kses_post( __( '<strong>Products</strong><script>alert(1);</script> and categories that inherit this global setting (i.e. they do not have a specific Google product category set) will use the new default immediately. Are you sure you want to proceed?', 'facebook-for-woocommerce' ) );
```

Comment:
whoops! Looks like some debug code sneaked in. The pitch also doesn't bold "Products" - was that just for testing?

Reply:
Thank you for catching that one @ChaseWiseman. Yes, the `<strong>` and the `<script>` tags were both for testing only. I remove them from the string in 6a1b21d4